packages/lambda-tiler/src/routes/attribution.ts

```diff
@@ -136,13 +136,14 @@ async function tileSetAttribution(
 
     items.push(item);
 
+    const providers = im.providers?.map((p) => ({ ...p, roles: p.roles ?? [] })) ?? getHost(host);
```

Comment:
We should fix the basemaps types to be correct, looking at stac-spec the roles are optional. https://github.com/radiantearth/stac-spec/blob/master/commons/common-metadata.md#provider